cog/chat.py

```
@@ -0,0 +1,388 @@
+        """
+
+        try:
+            connection, cursor = link_sql()
+            cursor.execute("SELECT nickname FROM chat_nick WHERE uid = %s", (user_id,))
+            ret = cursor.fetchall()
+            end(connection, cursor)
+            if ret and ret[0][0]:
+                return ret[0][0]
+        # 資料庫掛掉不該讓聊天功能跟著掛
+        # pylint: disable-next = broad-exception-caught
+        except Exception as exception:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)